Kickstarter-iOS/Views/Controllers/BackerDashboardViewController.swift

```diff
@@ -367,3 +367,11 @@ extension BackerDashboardViewController: UIGestureRecognizerDelegate {
       return true
   }
 }
+
+extension BackerDashboardViewController: TabBarControllerScrollable {
+  func scrollToTop() {
```

Comment:
Unfortunately due to the nature and setup of `BackerDashboardViewController` I didn't find a good way to test this.